src/rlpx/rlpx.ts

```diff
@@ -1,98 +1,126 @@
-const net = require('net')
-const os = require('os')
-const secp256k1 = require('secp256k1')
-const { EventEmitter } = require('events')
-const ms = require('ms')
-const Buffer = require('safe-buffer').Buffer
-const createDebugLogger = require('debug')
-const LRUCache = require('lru-cache')
-const pVersion = require('../../package.json').version
-const { pk2id, createDeferred } = require('../util')
-const Peer = require('./peer')
+import * as net from 'net'
+import * as os from 'os'
+import ms from 'ms'
+import { publicKeyCreate } from 'secp256k1'
+import { EventEmitter } from 'events'
+import { debug as createDebugLogger } from 'debug'
+import LRUCache from 'lru-cache'
+import { version as pVersion } from '../../package.json'
```

Comment:
The problem with this line is that the TS compiler is "smart":
if we want to compile src into dist the folder depth will change and the compiler will detect it. the path will either be invalid in the src file or the dist file so we have to pick one. It seems outDir and rootDirsettings are ignored to make sure the path works if we want to have a valid path in src.
A potential solution is to use a require statement:

// note: relative path only valid in .js file in dist
const { version : pVersion } = require('../../package.json')
or write code that actually conditionally reads and parses the file using fs.

Reply:
```suggestion
// note: relative path only valid in .js file in dist
const { version : pVersion } = require('../../package.json')
```